### ISSUE TITLE
RAI spamming - Purge Chunks

### ISSUE THREAD

**Issue by lucas2107** [VolmitSoftware/React#82]
Sometimes, RAI is spamming chunk purge without any reason (like X6 - X8) but is purging 0 chunks.

Server version: PaperSpigot 1.8.8

**Comment by cyberpwnn** [comment]
Is this only happening on paper 1.8? I have not seen this happen on other versions could you test?

**Comment by lucas2107** [comment]
I'm going to check with Spigot 1.8.8 and 1.9 tomorrow

**Comment by lucas2107** [comment]
http://prntscr.com/hso846

I'm alone on the server and moved from 40 blocks.
view distance=10

**Comment by cyberpwnn** [comment]
Is it paper only? I know this doesn't happen on spigot

**Comment by lucas2107** [comment]
I think It's only on paper, I can't test it atm :/

**Comment by Rbot48** [comment]
Same problem with Spigot 1.10.2:
```
[15:56:04 INFO]: [ASYNC] React: [RAI]: I'll fire Purge Chunks to help with chunk mass
[15:56:04 INFO]: React: [RAI]: Purging 447 chunks
[15:56:04 INFO]: React: [RAI]: Purged 6 chunks
[15:56:09 INFO]: React: [RAI]: Purging 441 chunks
[15:56:09 INFO]: React: [RAI]: Purged 0 chunks
[15:56:14 INFO]: React: [RAI]: Purging 441 chunks
[15:56:15 INFO]: React: [RAI]: Purged 0 chunks
[15:56:19 INFO]: React: [RAI]: Purging 441 chunks
[15:56:20 INFO]: React: [RAI]: Purged 0 chunks
[15:56:24 INFO]: React: [RAI]: Purging 441 chunks
[15:56:25 INFO]: React: [RAI]: Purged 0 chunks
[15:56:29 INFO]: React: [RAI]: Purging 441 chunks
[15:56:30 INFO]: React: [RAI]: Purged 0 chunks
[15:56:34 INFO]: React: [RAI]: Purging 441 chunks
[15:56:35 INFO]: React: [RAI]: Purged 0 chunks
[15:56:39 INFO]: React: [RAI]: Purging 441 chunks
[15:56:40 INFO]: React: [RAI]: Purged 0 chunks
[15:56:44 INFO]: React: [RAI]: Purging 441 chunks
[15:56:45 INFO]: React: [RAI]: Purged 0 chunks
[15:56:49 INFO]: React: [RAI]: Purging 441 chunks
[15:56:50 INFO]: React: [RAI]: Purged 0 chunks
[15:56:54 INFO]: React: [RAI]: Purging 441 chunks
```

**Comment by cyberpwnn** [comment]
Imposed a cooldown on rai for using purge chunks because paperspigot and sometimes spigot dont report unloading correctly. Fixed in 6.14742a

**Comment by Rbot48** [comment]
It's still purging 0 chunks:
```
[11:37:08 INFO]: React: [RAI]: Purging 441 chunks
[11:37:09 INFO]: React: [RAI]: Purged 0 chunks
[11:37:23 INFO]: React: [RAI]: Purging 441 chunks
[11:37:24 INFO]: React: [RAI]: Purged 0 chunks
[11:37:33 INFO]: React: [RAI]: Purging 441 chunks
[11:37:34 INFO]: React: [RAI]: Purged 0 chunks
[11:37:43 INFO]: React: [RAI]: Purging 441 chunks
[11:37:44 INFO]: React: [RAI]: Purged 0 chunks
[11:37:53 INFO]: React: [RAI]: Purging 441 chunks
[11:37:54 INFO]: React: [RAI]: Purged 0 chunks
[11:38:03 INFO]: React: [RAI]: Purging 441 chunks
[11:38:04 INFO]: React: [RAI]: Purged 0 chunks
[11:38:13 INFO]: React: [RAI]: Purging 441 chunks
[11:38:14 INFO]: React: [RAI]: Purged 0 chunks
[11:38:23 INFO]: React: [RAI]: Purging 441 chunks
[11:38:24 INFO]: React: [RAI]: Purged 0 chunks
[11:38:33 INFO]: React: [RAI]: Purging 441 chunks
[11:38:34 INFO]: React: [RAI]: Purged 0 chunks
[11:38:43 INFO]: React: [RAI]: Purging 441 chunks
[11:38:44 INFO]: React: [RAI]: Purged 0 chunks
[11:38:53 INFO]: React: [RAI]: Purging 441 chunks
[11:38:54 INFO]: React: [RAI]: Purged 0 chunks
[11:39:03 INFO]: React: [RAI]: Purging 441 chunks
[11:39:04 INFO]: React: [RAI]: Purged 0 chunks
[11:39:13 INFO]: React: [RAI]: Purging 441 chunks
[11:39:14 INFO]: React: [RAI]: Purged 0 chunks
[11:39:28 INFO]: React: [RAI]: Purging 441 chunks
[11:39:29 INFO]: React: [RAI]: Purged 0 chunks
[11:39:38 INFO]: React: [RAI]: Purging 441 chunks
[11:39:39 INFO]: React: [RAI]: Purged 0 chunks
...
```
Spigot 1.10.2
React 6.14742a

**Comment by cyberpwnn** [comment]
I really do hate paperspigot

**Comment by cyberpwnn** [comment]
Fixed in 6.15201a
